planning/behavior_path_planner/src/utilities.cpp

```diff
@@ -1034,14 +1034,7 @@ bool setGoal(
 
 const Pose refineGoal(const Pose & goal, const lanelet::ConstLanelet & goal_lanelet)
 {
-  // return goal;
   const auto lanelet_point = lanelet::utils::conversion::toLaneletPoint(goal.position);
-  const double distance = boost::geometry::distance(
-    goal_lanelet.polygon2d().basicPolygon(), lanelet::utils::to2D(lanelet_point).basicPoint());
-  if (distance < std::numeric_limits<double>::epsilon()) {
-    return goal;
-  }
-
```

Comment:
Let me confirm.
Is this function left for use in other part?

Reply:
Currently No, so this modification does not affect. 
But I want to use it later in https://github.com/autowarefoundation/autoware.universe/pull/2335, so left it.

Reply:
It is okay to left this if you have a purpose.
Thank you.